src/applications/vaos/sass/vaos.scss

```diff
@@ -4,7 +4,7 @@
 .vaos-landing__icon {
   display: inline-block;
   font-size: 19px;
-  color: white;
```

Comment:
Had to change this from `white` to `#fff` to quiet the `no-color-keywords` error. But we still get a `no-color-literals` error. So, what do we do? Make a variable named `$white` to make all the rules happy? Seems pretty silly.

Reply:
maybe $color-white: https://design.va.gov/design/color-palette

Reply:
haha, yeah, I noticed that after your Slack comment. 🤦‍♂